widgetssdk/src/main/java/com/glia/widgets/callvisualizer/CallVisualizerActivityWatcher.kt

```diff
@@ -94,15 +95,15 @@ internal class CallVisualizerActivityWatcher(
         activityLauncher.launchWebBrowser(activity, title, url)
     }
 
-    private fun showSnackBar(activity: Activity) {
-        makeSnackBar(activity).show()
-    }
+    private fun showTimedOutSnackBar(activity: Activity) = showSnackBar(activity, R.string.engagement_incoming_request_timed_out_message)
+
+    private fun showAlreadyInCallSnackBar(activity: Activity) = showSnackBar(activity, R.string.entry_widget_call_visualizer_description)
```

Comment:
At the moment this localization fits our needs here. I'm thinking if an integrator might need different wording for an ongoing Call Visualizer and any ongoing engagement when using an Engagement Launcher? 🤔

Reply:
This snack bar is only used in cases of ongoing CVs, so the use case is pretty much the same.
I'll rename `showAlreadyInCallSnackBar(activity: Activity)` -> `showAlreadyInCVSnackBar(activity: Activity)`